overlord/devicestate/devicemgr.go

```diff
@@ -353,7 +361,10 @@ func (m *DeviceManager) ensureSeedYaml() error {
 		return nil
 	}
 
-	tsAll, err := populateStateFromSeed(m.state)
+	var tsAll []*state.TaskSet
+	timings.Run(perfTimings, "state-from-seed", "populate state from seed", func(timings.Measurer) {
+		tsAll, err = populateStateFromSeed(m.state)
```

Comment:
can be in a follow up but I do think we do need to go into populateStateFromSeed and for example measure the DeriveSideInfo stuff separately

Reply:
Sure, will do in a followup.